src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,61 @@
+import React, {useEffect, useState} from 'react';
+import {Link, useParams} from "react-router-dom";
+import './Subreddit.css';
+import axios from "axios";
+
+function Subreddit() {
+    const [data, setData] = useState([]);
+    const {id} = useParams();
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const result2 = await axios.get(`https://www.reddit.com/r/${id}/about.json`);
+                console.log(result2);
+                setData(result2.data.data);
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        if (id) {
+            fetchData();
+
+        }
+    }, [id]);
+
+    return (
+        <>
+            <body>
+
+            <header className="container" id="top-section-subreddit">
+                <h1 className="top-section-title">r/{id}</h1>
+                <h4>Subreddit specifications</h4>
+            </header>
+
+            <main>
+                <section className="container" id="mid-section-subreddit">
+
+                    {Object.keys(data).length > 0 && (
```

Comment:
Je check kan hier dan ook een stuk simpeler
```suggestion
                    {data && (
```